src/Microsoft.AspNet.Mvc.Core/ModelBinding/CompositeModelBinder.cs

```diff
@@ -148,6 +148,7 @@ public CompositeModelBinder([NotNull] IEnumerable<IModelBinder> modelBinders)
                 Model = oldBindingContext.Model,
                 ModelMetadata = oldBindingContext.ModelMetadata,
                 ModelName = modelName,
+                FieldName = oldBindingContext.FieldName,
```

Comment:
This is the one location in code where we still have ad-hoc creation of an MBC that doesn't really follow an established pattern. My next changes will remove it.